### ISSUE TITLE
Duplicate data properties 

### ISSUE THREAD

**Issue by jziemer2** [edmcouncil/fibo#1018]
Pertains to https://spec.edmcouncil.org/fibo/ontology/master/latest/dev.ttl.zip
The list at the bottom shows duplicate Data Properties with an incrementing number at the end.

For example, CorporateActionsEvents.ttl:
```
fibo-cae-ce-cae:mandatory
	a owl:DatatypeProperty ;
	rdfs:label "mandatory"@en ;
	rdfs:comment "QName: fibo-cae-ce-cae:mandatory" ;
	rdfs:domain fibo-cae-ce-cae:BonusRightsIssue ;
	rdfs:isDefinedBy <https://spec.edmcouncil.org/fibo/ontology/CAE/CorporateEvents/CorporateActionsEvents/> ;
	rdfs:range xsd:boolean ;
	.

fibo-cae-ce-cae:mandatory.1
	a owl:DatatypeProperty ;
	rdfs:label "mandatory"@en ;
	rdfs:comment "QName: fibo-cae-ce-cae:mandatory.1" ;
	rdfs:domain fibo-cae-ce-cae:IssuerCorporateAction ;
	rdfs:isDefinedBy <https://spec.edmcouncil.org/fibo/ontology/CAE/CorporateEvents/CorporateActionsEvents/> ;
	rdfs:range xsd:boolean ;
	skos:definition "Whether the action is mandatory on the Holder."@en ;
	.

fibo-cae-ce-cae:mandatory.2
	a owl:DatatypeProperty ;
	rdfs:label "mandatory"@en ;
	rdfs:comment "QName: fibo-cae-ce-cae:mandatory.2" ;
	rdfs:domain fibo-cae-ce-cae:MandatoryCashDividendEvent ;
	rdfs:isDefinedBy <https://spec.edmcouncil.org/fibo/ontology/CAE/CorporateEvents/CorporateActionsEvents/> ;
	rdfs:range xsd:boolean ;
	skos:definition 'Whether the action is mandatory on the Holder. This is "yes" i.e. it is a mandatory event.'@en ;

```

fibo-cae-ce-cae:mandatory.1
fibo-cae-ce-cae:mandatory.2
fibo-cae-ce-cae:mandatory.3
fibo-cae-ce-cae:requiresResponse.1
fibo-civ-fnd-civ:description.1
fibo-civ-fnd-civ:name.1
fibo-civ-fnd-civ:subFund.1
fibo-der-der-fwd:cashDelivery.1
fibo-der-der-fwd:deliveryDate.1
fibo-der-etd-eto:strikePrice.2
fibo-fnd-plc-adr:hasAddressLine1
fibo-fnd-plc-adr:hasAddressLine2
fibo-fnd-plc-adr:hasAddressLine3
fibo-loan-tem-sta:delinquent.1
fibo-loan-typ-sec:secured.1
fibo-sec-dbt-ab-cdo:isCash.1
fibo-sec-dbt-ab-cdo:seniority.1
fibo-sec-dbt-ab-cdo:seniority.2
fibo-sec-dbt-ab-cdo:seniority.3
fibo-sec-dbt-ab-cdo:seniority.4
fibo-sec-dbt-ab-mbs:passThrough.1
fibo-sec-dbt-ab-mbs:passThrough.2
fibo-sec-dbt-ab-mbs:passThrough.3
fibo-sec-dbt-ab-mbs:passThrough.4

**Comment by ElisaKendall** [comment]
The 3 address line properties are not a mistake - these are used typically for address mapping to back end databases at a number of banks.  

The others were generated from the original models in 2013-2014 when we first converted them from EA and Excel to OWL, and are in provisional ontologies.  We've eliminated hundreds of them, and these are in ontologies that have not been addressed to date.  There are other duplicates that don't have these .x number suffixes, but are also still in provisional ontologies that should also be addressed over time.
